package.json

```diff
@@ -26,9 +26,10 @@
   },
   "dependencies": {
     "@nuxt/content": "^1.9.0",
-    "@nuxtjs/pwa": "^3.3.5",
-    "dateformat": "^4.5.0",
-    "core-js": "^3.8.3",
+    "@nuxtjs/pwa": "^3.0.2",
```

Comment:
You should use git pull before u adds something. Dependbot has upgraded these dependencies, but now, u want to downgrade them.